packages/react-components/react-nav-preview/src/components/NavDrawer/useNavDrawer.ts

```diff
@@ -38,6 +38,7 @@ export const useNavDrawer_unstable = (props: NavDrawerProps, ref: React.Ref<HTML
     root: slot.always<DrawerProps>(
       {
         ref,
+        role: 'menu',
```

Comment:
I'd actually strongly recommend against using `menu` semantics in a navigation use case like this, for several reasons.

The biggest one is the basic issue that menu roles obscure link information and prevent built-in link behavior for ATs that use semantics (primarily screen readers, also voice control). This means:
- no checking the URL before following the link
- will not show up in lists of links, using screen reader UI (this is a huge issue for navigation, this can be a big time saver when people know where they want to go)
- will not show up using link-specific virtual cursor shortcuts
- potential issues with link-specific features built into Voice Control

The secondary problem is that menus are not conducive to virtual cursor navigation in general, beyond just not working with the link shortcut. Given how high-importance navigation is, IMO this means we should never use `menu` roles as the default for a set of controls intended to work as any type of navigation (whether primary or auxiliary).

Reply:
The drawback of course is that link/button semantics do not auto-switch the user into application mode, but I've generally found that's not really an issue, and in cases where the entire UI is not already wrapped in `role=application`, folks generally prefer to use the built-in screen reader commands.

If the entire UI is already wrapped in `role=application`, then this wouldn't be a problem anyway, and link/button semantics would still be best 😄